### DISCUSSION TITLE
ipam-coupling: PTR records don't create

### DISCUSSION THREAD

**Discussion by ar-reser** [sys4/netbox-plugin-dns#170] [Q&A] [answered]
ipam coupling creates only A Record. PTR record do not pop up automatically.

![Screenshot 2024-02-09 152830](https://github.com/peteeckel/netbox-plugin-dns/assets/159555729/7f1ea8e2-7b36-4a7b-8d4d-b1125dc231f2)
![Screenshot 2024-02-09 160128](https://github.com/peteeckel/netbox-plugin-dns/assets/159555729/35314fac-072c-4117-bd89-ccc9c3124321)
<img width="817" alt="Screenshot 2024-02-09 161245" src="https://github.com/peteeckel/netbox-plugin-dns/assets/159555729/d1389b95-a9ad-4948-9af5-ff9f22911589">

NetBox version: 3.7.0
Python version: 3.9.18
NetBox DNS version: 0.21.14

**Comment by peteeckel** [comment]
Have you created zone `249.31.172.in-addr.arpa`?

Also are you aware that `netbox-plugin-dns 0.21.14` is not the latest release? Shouldn't be critical in your case, though.

**Reply by ar-reser** [reply, marked as answer]
Thank you for your fast reply! 

The zone was created but without '.in-addr.arpa' at the end. After switching the zone name to  '31.172.in-addr.arpa' everything works as expected.
<img width="825" alt="image" src="https://github.com/peteeckel/netbox-plugin-dns/assets/159555729/50116b2f-8fa5-40f8-b0ee-30e71ec29b5a">

Also thanks for the comment about the new version, update is sceduled

**Comment by peteeckel** [comment]
Glad it worked out. 

By the way, are you using the domain `cae.lan` in production? That's dangerous territory as people using AVM routers, which traditionally used the domain `fritz.box` for management purposes, found out recently when the TLD `.box` was created and registered, leading to all kinds of strange behaviour.

Currently, there is no such TLD, so you're fine for the moment. But there also isn't an RFC that reserves `.lan` for special purposes, so sooner or later using it internally may spell trouble.
